### ISSUE TITLE
new UI analytics tab - blank white page

### ISSUE THREAD

**Issue by okaikov** [optuna/optuna-dashboard#462]
### Description

in the new UI analytics tab, getting a blank white page
in the optuna dashboard server I don't see any errors

### How to Reproduce

1. Optuna's objective function is '...'.
2. Run optuna-dashboard with '...'
3. Open '...' page, then click '...'.
4. An error occurs.

### Python version

3.8.1

### Optuna version

optuna==3.1.1

### optuna-dashboard version or git revision

optuna-dashboard==0.9.2

### Web browser

Google Chrome, Firefox, etc.

**Comment by c-bata** [comment]
Thank you for reporting. Could you share the minimal reproducible code with us?

**Comment by okaikov** [comment]
I have a rather complex code which I can't share. and I'm unable to reproduce  on a small example.
is there a way to printout any debug logs in the dashboard that point us to the issue?

**Comment by c-bata** [comment]
@okaikov We've addressed a similar problem reported in Issue #410. However, as we don't have a clear reproduction method for the issue you reported, we're uncertain if it's been fully resolved. Therefore, we're going to close this issue for now. 

If you still encounter this problem after the upcoming release of v0.10.0, please help us by providing an example that we can use to reproduce the issue and open a new issue for it. Without a reproducible example, it becomes almost impossible for us to fix the problem. We appreciate your understanding.

**Comment by okaikov** [comment]
The issue is now fixed.
Thx alot! :)

**Comment by ZWL-S** [comment]
I met the same issue, with the example code provided in the Getting Started

```
import optuna

def objective(trial):
    x = trial.suggest_float("x", -100, 100)
    y = trial.suggest_categorical("y", [-1, 0, 1])
    return x**2 + y

if __name__ == "__main__":
    study = optuna.create_study(
        storage="sqlite:///db.sqlite3",  # Specify the storage URL here.
        study_name="quadratic-simple"
    )
    study.optimize(objective, n_trials=100)
    print(f"Best value: {study.best_value} (params: {study.best_params})")
```
and starting the dashboard with

`optuna-dashboard sqlite:///db.sqlite3`

The analytics tab of the project will flash and then turn into a black page.

Here is my env:
**Python version**
3.8.20

**Optuna version**
optuna==4.0.0

**optuna-dashboard version or git revision**
optuna-dashboard==0.16.2

**Web browser**
Google Chrome
Version 130.0.6723.70 (Official Build) (arm64)

**OS**
Mac (But met the same issue on Windows as well)

**Comment by thomasroodnl** [comment]
I am having the same issue as @ZWL-S, Python 3.12.7, but otherwise the same package versions. I am on Windows.

It seems the analytics tab works as long as there is one datapoint/trial result, but once there are more it will go to a blank screen. The only hint I could find is a JavaScript error in the browser console: 

![image](https://github.com/user-attachments/assets/c4d12066-12b0-4ea2-abed-358d27b90d51)

Would love to see if this could be reopened and solved, I'd be happy to help narrow down the issue.

**Comment by c-bata** [comment]
This bug seems to be duplicated with https://github.com/optuna/optuna-dashboard/issues/978. Let me close this and continue working on #978.

**Comment by thomasroodnl** [comment]
Can confirm it is fixed in v0.17.0, thank so much for the quick fix :)
